docker-compose.yml

```diff
@@ -1,11 +1,22 @@
-version: '3.7'
+version: "3.7"
 
 services:
+  influxdb:
+    image: influxdb:latest
```

Comment:
latest is generally not the best option ;)

Especially here - influx 2.0 was recently released. Release 2.0 changes a lot and will most likely be not compatible with current setup. Additionally, I assume `latest` will soon point to influxdb 2.0